lib/ReactTags.js

```diff
@@ -56,6 +56,42 @@ class ReactTags extends React.Component {
     this.setState({ query })
   }
 
+  handlePaste (e) {
+    // allow over-ride, if there is a need
+    if (this.props.onPaste) {
+      return this.props.onPaste(e)
+    }
+
+    const { delimiterChars } = this.props
+
+    e.preventDefault()
+
+    // get the text data from the clipboard
+    const data = e.clipboardData.getData('Text')
+
+    if (data && delimiterChars.length > 0) {
+      // split the string based on the delimiterChars as a regex
+      const tags = data.split(new RegExp(delimiterChars.join('|'), 'g'))
```

Comment:
Okay, I'll do this. Just one thing, is that we may find this causes inconsistent behaviour WRT to handleKeyDown, since the logic there won't deal with multi-character delimiters. I'll compromise and add a note in the readme.